src/org/zaproxy/clientapi/gen/Core.java

```diff
@@ -532,6 +532,21 @@ public ApiResponse setOptionUseProxyChainAuth(String apikey, boolean bool) throw
 		return api.callApi("core", "action", "setOptionUseProxyChainAuth", map);
 	}
 
+	public ApiResponse deleteSiteNode(String apikey, String url, String method) throws ClientApiException {
+		Map<String, String> map = null;
+		map = new HashMap<String, String>();
```

Comment:
These can be combined I believe...

I realize it's reused code, but map = new HashMap should be fine without first being null.